projects/freetype2/Dockerfile

```diff
@@ -16,13 +16,13 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER mike.aizatsky@gmail.com
-RUN apt-get update && apt-get install -y make autoconf libtool libarchive-dev
+RUN apt-get update && apt-get install -y autoconf cmake libarchive-dev libtool make
 
 # Get some files for the seed corpus
 ADD https://github.com/adobe-fonts/adobe-variable-font-prototype/releases/download/1.001/AdobeVFPrototype.otf $SRC/font-corpus/
 RUN git clone https://github.com/unicode-org/text-rendering-tests.git && cp text-rendering-tests/fonts/* $SRC/font-corpus
```

Comment:
I left these two samples sources in here; they don't hurt anyone and they don't really fit into the "new way" of approaching FreeType's fuzzers (yet).